GLRenderer/GLRenderer/Textures/TextureLoader.cpp

```diff
@@ -24,6 +24,7 @@ std::shared_ptr<GLRenderer::Textures::C_Texture> TextureLoader::LoadAndInitTextu
 	GLenum	Target = GL.translate(Texture.target());
```

Comment:
Last file to use "old" API?

Reply:
Not really the last one, but I am getting there. Right now I know about ray tracing and that is it.